<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadItem</class>
 <widget class="QWidget" name="DownloadItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="1">
    <widget class="QWidget" name="widgetBackground" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widgetBackground
{
background-color: #E1E1E1;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="NormalLabel" name="labelWarning">
          <property name="minimumSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource/resource.qrc">:/download/download/event.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="NormalLabel" name="labelName">
          <property name="text">
           <string>Imagexxx</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButtonClose">
           <property name="styleSheet">
            <string notr="true">border: 0px;
color: #4A4A4A;</string>
           </property>
           <property name="icon">
            <iconset resource="../resource/resource.qrc">
             <normaloff>:/download/download/delete1.png</normaloff>:/download/download/delete1.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="styleSheet">
         <string notr="true">QProgressBar
{
max-height: 10px;
border: 0px;
border-radius: 5px;
background-color: #d1d1d1;
}
QProgressBar::chunk[error=&quot;false&quot;]
{
background-color: #39b8e4;
border-radius: 5px;
}
QProgressBar::chunk[error=&quot;true&quot;]
{
background-color: #e64b4a;
border-radius: 5px;
}</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="error" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="NormalLabel" name="labelValue">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>100%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NormalLabel</class>
   <extends>QLabel</extends>
   <header>normallabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
